packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.tsx

```diff
@@ -160,7 +160,7 @@ export const CreateAlertDefinition = () => {
     return (
       notificationChannelWatcher
         .map((id) =>
```

Comment:
```suggestion
  return (
      notificationData?.filter((notification) =>
        notificationChannelWatcher.includes(notification.id)
      ) ?? []
    );
```

can we achieve similarly like this in single iteration